src/main/resources/resource/WebGui/app/views/tabsViewCtrl.js

```diff
@@ -33,12 +33,12 @@ angular.module('mrlapp.mrl').controller('tabsViewCtrl', ['$location','$scope', '
         $scope.panels = panels
 
         if (!$scope.view_tab && panels.length > 0 && $scope.panels[$scope.panels.length - 1].name.startsWith('intro')) {
```

Comment:
No body in this if statement.. dead code.

Reply:
heh ... you never go here :)
There's lots of bigger fish to fry in these javascript parts